middleware/auth/jwt/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/go-kratos/kratos/auth/jwt/v2
+
+go 1.16
+
+require (
+	github.com/go-kratos/kratos/v2 v2.0.5
+	github.com/golang-jwt/jwt v3.2.1+incompatible
+	github.com/stretchr/testify v1.7.0
+)
+
+replace github.com/go-kratos/kratos/v2 => ../../../
```

Comment:
should it be `../../../../kratos` to avoid accidental dependence

Reply:
ok.3q